### PR TITLE
[Win Pipeline] Fix Multiselect Undo/Redo

### PR DESCRIPTION
**Pull request by cra0zy**

### CODE REVIEW

**Comment by tomspilman** [comment]
@jamesford42 - Can you review this change?

File: Tools/Pipeline/Windows/MainView.cs [hunk 1]
```diff
@@ -666,9 +662,16 @@ private void TreeViewAfterSelect(object sender, TreeViewEventArgs e)
             _controller.Selection.Clear(this);
             _propertyGrid.SelectedObject = null;
 
+            _oldValues.Clear();
+
             foreach (var node in _treeView.SelectedNodes)
             {
-                _controller.Selection.Add(node.Tag as IProjectItem, this);
```
Comment (jamesford42) [inline Tools/Pipeline/Windows/MainView.cs]: This is a test.

File: Build/Projects/Pipeline.definition [hunk 2]
```diff
@@ -69,8 +71,6 @@
     <Compile Include="Common\PipelineTypes.cs" />
     <Compile Include="Common\Extensions.cs" />
 
-    <Compile Include="Windows\UpdateAction.cs" />
```
Comment (jamesford42) [inline Build/Projects/Pipeline.definition]: This is a test2

File: Tools/Pipeline/Common/PipelineController.UpdateContentItemAction.cs [hunk 3]
```diff
@@ -0,0 +1,53 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
```
Comment (jamesford42) [inline Tools/Pipeline/Common/PipelineController.UpdateContentItemAction.cs]: This is a test3

File: Tools/Pipeline/Windows/MainView.cs [hunk 4]
```diff
@@ -160,16 +161,11 @@ private void OnPropertyGridPropertyValueChanged(object s, PropertyValueChangedEv
             if (args.ChangedItem.Label == "References")
                 _controller.OnReferencesModified();
 
-            // TODO: This is the multi-select case which needs to be handled somehow to support undo.
-            if (args.OldValue == null)
-                return;
-
-            var obj = _propertyGrid.SelectedObject;
+            var obj = _oldValues.Count == 1 ? _propertyGrid.SelectedObject : new ContentItem();
```
Comment (jamesford42) [inline Tools/Pipeline/Windows/MainView.cs]: This could be written so that a `new ContentItem` we don't actually use is not being allocated.

**Comment by cra0zy** [comment]
Comments @jamesford42 made and the old code can be found: https://github.com/cra0zy/MonoGame/commit/e60ad4f33435ecfdcbf47077e3bfbe53f91cf809

@jamesford42 Next time please comment by clicking on "Files changed" tab and commenting there instead on commenting on a commit message, I like to do a rebase when I have to modify something instead of adding another commit.

Does this PR pass now?

**Comment by tomspilman** [comment]
> Next time please comment by clicking on "Files changed" tab

Yeah... we were trying to figure out what he did there.  It confused the hell out of us as to what went wrong.

**Comment by tomspilman** [comment]
Investigating why we're not seeing any build bot notifications in this.  But yeah this should get merged it a bit.

**Comment by tomspilman** [comment]
Ok fixed... thanks @cra0zy!  Merging!

**Pull request merged**